app/controllers/sufia/admin/strategies_controller.rb

```diff
@@ -0,0 +1,9 @@
+module Sufia
+  module Admin
+    class StrategiesController < Flip::StrategiesController
+      before_action do
+        authorize! :manage, Feature
```

Comment:
But aren't both of the new controllers in the `Sufia::Admin` namespace? I don't know why you'd use the explicit `Sufia::` namespace in one and not the other.